apiary-metastore-events/kafka-metastore-events/kafka-metastore-receiver/README.md

```diff
@@ -0,0 +1,34 @@
+# Apiary Metastore Events
+
+## Overview
+
+This module contains classes for retrieving and processing Hive Metastore events from Kafka.
+
+## Kafa Metastore Receiver Configuration
+
+To configure the receiver, a builder is provided. The following parameters are available:
```

Comment:
So, do we allow the users to create a Properties object where the below are the keys which point to the config values? Or does the builder just produce this thing which in turns gets handed over to some Kafka API? I'm just a bit confused if the user would actually use the properties below or if they would just programmatically call the Builder and not know about these internals?

Reply:
The builder does indeed create a Properties object, and all of the defaults are the same as the defaults in the Kafka library itself. So the user doesn't have to do anything really. But from what I've seen Kafka doesn't provide a library/builder to do something like this so it would be quite manual. 

I'm guessing that @abhimanyugupta07 picked out some of the common/more likely to change properties? So the user of this library can do whatever they want and insert their own Properties in the constructor, or they can use the builder provided here which is a little more fluent for discovering what can be changed. And the readme is there just for reference.

What do you think?

Reply:
Saw this through @abhimanyugupta07's fresh pair of eyes, going to refactor this to be a builder for this class, not the properties themselves. It looks alot nicer that way.